tools/ghsa/convert_ghsa.py

```diff
@@ -185,19 +186,23 @@ def get_affected(ghsa: Dict[str, Any]) -> List[Dict[str, Any]]:
         if ecosystem in NAME_NORMALIZER:
             name = NAME_NORMALIZER[ecosystem](name)
 
+        db = {
+            # Attribution.
+            'ghsa': ghsa['permalink'],
+            'cwes': cwes,
+        }
+
+        if include_cvss:
+            db['cvss'] = cvss
```

Comment:
We can actulaly probably just remove these completely. The CVSS score is now part of the OSV schema is no longer necessary here.

Reply:
Done!